desktop/main.ts

```diff
@@ -374,6 +388,39 @@ const mainWindow = (): Promise<void> => {
                         });
                     });
                 });
+
+                ipcMain.handle(ELECTRON_EVENTS.CHECK_LOCATION_PERMISSION, async () => {
+                    try {
+                        type MacPermissionsModule = {
+                            getAuthStatus?: (authType: AuthType) => PermissionType | 'not determined';
+                        };
+                        const macPermissionsModule = (await import('node-mac-permissions')) as MacPermissionsModule | {default: MacPermissionsModule};
```

Comment:
Do we need to import this every time the event callback is called?

I think it would make more sense to lift `macPermissionsModule` to the global scope and either:

- synchronously import it once on app startup (probably simplest, and it's unlikely it would have a big impact on TTI, but you'd have to measure 🤷🏼), or
- check if it's defined in the callback, and only if it's not then dynamically import it, but only once per app startup, not once per callback execution

Reply:
Also, I'm not sure what kind of tree-shaking we have in place for the Electron main process, but if it's possible to import _only_ `getAuthStatus` that would probably be better than importing the whole module.